### ISSUE TITLE
[Feature Request] Formatting codes?

### ISSUE THREAD

**Issue by samwfowler** [andrewphorn/ClassiCube-Client#259]
Not entirely sure about this one- but I think it would be nice to be able to add some formatting to chat when sending messages- maybe underlining text e.t.c. Just a speculation.

**Comment by 0sean** [comment]
This'd be cool! And maybe a way for servers to manage it?

**Comment by samwfowler** [comment]
Yes- it could work just like custom colors- and with the extended character map available it shouldn't be too limited
